src/instructlab/data/generate_data.py

```diff
@@ -82,21 +178,26 @@ def gen_data(
     client = openai.OpenAI(
         base_url=api_base, api_key=api_key, http_client=http_client(http_client_params)
     )
+    # Third Party
+    from instructlab.sdg.generate_data import generate_data
+
+    # pylint: disable=ungrouped-imports
+    from instructlab.sdg.utils import GenerateException
 
     try:
         logger.info(
-            f"Generating synthetic data using '{pipeline}' pipeline, '{model_path}' model, '{taxonomy_path}' taxonomy, against {api_base} server"
+            f"Generating synthetic data using '{pipeline}' pipeline, '{model_name}' model, '{taxonomy}' taxonomy, against {api_base} server"
```

Comment:
Arg rename?

Reply:
same as above

Reply:
Ack, will resolve this but see my point in that thread.